server/service/src/purchase_order_line/insert/mod.rs

```diff
@@ -62,23 +77,33 @@ pub fn insert_purchase_order_line(
             let validate_input = validate::ValidateInput {
```

Comment:
This doesn't need to be here anymore no? Can just pass the whole input into the validate method...